lib/pages/login-page.js

```diff
@@ -43,6 +43,40 @@ export default class LoginPage extends BaseContainer {
 		} );
 	}
 
+	use2FAMethod( twoFAMethod ) {
```

Comment:
On a "normal flow" test, do we even need to call that?
Since the user will be redirected to whatever method of 2FA she has, isn't she?
What am I missing here?

Reply:
No, users might have all 2fa methods enabled and might see the push screen once and the sms screen another time, I created this method so we would be sure to be on the right page, and if we're not that we are offered the options to switch to the right page.